src/linear_algebra/affine_space/combination.lean

```diff
@@ -296,6 +296,72 @@ end
 
 end finset
 
+namespace finset
+
+variables (k : Type*) {V : Type*} {P : Type*} [division_ring k] [add_comm_group V] [module k V]
+variables [affine_space V P] {ι : Type*} (s : finset ι)
+
+/-- The weights for the centroid of some points. -/
+@[nolint unused_arguments]
+def centroid_weights (i : ι) := (finset.card s : k) ⁻¹
```

Comment:
Wouldn't it be clearer to announce `centroid_weights : ι → k` and define it using a lambda (or `function.const` if you prefer, either way make it clear you are defining a constant function).

Reply:
Changed to use `function.const` in the definition.